builder/qemu/builder.go

```diff
@@ -154,8 +154,16 @@ func (b *Builder) Prepare(raws ...interface{}) ([]string, error) {
 		if runtime.GOOS == "windows" {
 			b.config.Accelerator = "tcg"
 		} else {
-			b.config.Accelerator = "kvm"
+			if fp, err := os.Open("/dev/kvm"); err != nil {
```

Comment:
/dev/tcg not exists always. this is qemu tiny code generator.

Also /dev/kvm may be present but not works, so we need to dry to open it.
If user load kvm-intel or kvm-amd /dev/kvm presented and works fine. If user load kvm only module without kvm-intel or kvm-amd this device may not work. Also use (in caose of docker or static devicie nodes) can have /dev/kvm via mknod but it not works and if you try to open it - you have error.
I'm test this under native kvm system and docker. I does not see any problems..